nservicebus/msmq/troubleshooting.md

```diff
@@ -69,6 +69,44 @@ It is recommended to have batch setup scripts that run on server startups to con
 While non-transactional messaging in a Network Load Balancing (NLB) environment is possible, it is much harder to achieve load-balancing in transactional MSMQ. [Microsoft provides a detailed answer](https://support.microsoft.com/en-us/kb/899611).
 
 
+## MSMQ messages disappear from outbound queue but never arrive in the inbound queue
+
+Message are expected to appear in the destination queue. When MSMQ is stopped at the receiver, the messages pile up in outgoing queues at the sender. When restarting MSMQ on the receiver all messages are gone from the outgoing queues, but they are not visible at the receiver.
+
+
+### Resolution
```

Comment:
these are not resolutions. they are more path to take to find the problem. so i think the title should change